runner/runner.go

```diff
@@ -87,6 +90,7 @@ func (r *Runner) Run(ctx context.Context) error {
 func (r *Runner) runListener(ctx context.Context) error {
 	if isFileExists(r.startedFlagFile) {
 		metrics.UpdateRunnerPodState(metrics.Stale)
+		fmt.Println("Pod is stale; waiting for deletion")
```

Comment:
Why don't you use a logger?

Reply:
The entire runner did not have a Logger installed, so i modified it to use "sigs.k8s.io/controller-runtime/pkg/log/zap".